src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -227,6 +226,9 @@ else if (request.getParameter("playGame") != null){
 				startGame(request, response);
 				// response.sendRedirect("/admin");
 			}
+			else if(request.getParameter("clearConversations") != null){
```

Comment:
Did you add a method for admins to clear all conversations? I'm confused about what exactly this does.

Reply:
Sorry so this actually does nothing, I got fed up with creating so many new conversations that I started making the clearConversations function but then never actually went deep into the datastore to actually clear them. I'll delete this in a new commit.